### PR TITLE
Changed OpenRPC Document to be one file instead of many

### LINKED ISSUE #73: Use ethereum execution api spec
should be reffing https://raw.githubusercontent.com/ethereum/execution-apis/assembled-spec/refs-openrpc.json as the base spec for Ethereum

### PR DESCRIPTION
**Pull request by shanejonas**
This simplifies the build and contributions

fixes #73

### CODE REVIEW

**Comment by shanejonas** [comment]
@SocketSecurity ignore jest-cli@29.3.1

**Comment by shanejonas** [comment]
@SocketSecurity ignore jest@29.3.1

File: tests/index.test.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import {OpenrpcDocument} from "@open-rpc/meta-schema"
+import mm from '../dist/build/openrpc.json';
+
+const MetaMaskOpenRPCDocument = mm as OpenrpcDocument;
+
+// openrpc 1.0.0 => string
```
Comment (shanejonas) [inline tests/index.test.ts:6]:
```suggestion
```

**Review by BelfordZ — APPROVED** [review]

**Comment by BelfordZ** [comment]
lgtm

**Pull request merged**